CatAnalyzer/python/csvWeights_cfi.py

```diff
@@ -5,6 +5,15 @@
     leptons = cms.InputTag(""), ## Selected leptons to apply overlap removal
     minPt = cms.double(20),
     maxEta = cms.double(2.41),
-    csvSFHF = cms.string("csv_rwt_fit_hf_v2_final_2017_6_7_all.root"),
-    csvSFLF = cms.string("csv_rwt_fit_lf_v2_final_2017_6_7_all.root"),
+    csvSFHF = cms.string("csv_rwt_fit_hf_v0_final_2018_2_13.root"),
+    csvSFLF = cms.string("csv_rwt_fit_lf_v0_final_2018_2_13.root"),
+)
+
+deepcsvWeights = cms.EDProducer("CSVWeightProducer",
+    src = cms.InputTag("catJets"),
+    leptons = cms.InputTag(""), ## Selected leptons to apply overlap removal
+    minPt = cms.double(20),
+    maxEta = cms.double(2.41),
+    deepcsvSFHF = cms.string("Deepcsv_rwt_fit_hf_v2_final_2018_2_12test.root"),
+    deepcsvSFLF = cms.string("Deepcsv_rwt_fit_lf_v2_final_2018_2_12test.root"),
```

Comment:
You will see warning messages that you have to add only one module in the cfi files.
Give different name such as csvWeights_cff.py or add new file deepcsvWeights_cfi.py and make them to contain only one module for each.

Reply:
Made another file, deepcsvWeights_cfi.py .